cmake/Modules/CPM.cmake

```diff
@@ -1,6 +1,8 @@
-set(CPM_DOWNLOAD_VERSION c3c7e2d9a3b890511891ffc415d7ea23c689068a) # 0.27.3
+set(CPM_DOWNLOAD_VERSION 0.28.4)
```

Comment:
We use a git commit hash and the raw link to make sure that the upstream tag doesn't change out from underneath us. Any chance you could update this to follow that pattern?

Reply:
Finished.
Thanks for your comment : )